packages/desktop-electron/package.json

```diff
@@ -88,9 +85,9 @@
     "npmRebuild": false
   },
   "dependencies": {
+    "better-sqlite3": "^9.6.0",
     "electron-is-dev": "2.0.0",
     "electron-log": "4.4.8",
-    "loot-core": "*",
```

Comment:
This was causing us to import all of loot-cores dependencies. We don't need to do this we bundle it - we have no need for loot-core's node_modules.